draft-group-tls-external-psk-guidance.md

```diff
@@ -231,15 +231,23 @@ compromised `A`, the attacker can successfully impersonate `B` to `A` using this
 
 # Recommendations for External PSK Usage
 
-To achieve the security goals from {{sec-properties}} when the external PSK will not be combined with a pairwise secret value,
-applications MUST use external PSKs that adhere to the following requirements:
 
-- Each PSK MUST NOT be shared between with more than two logical nodes. As a result, an agent
+Applications MUST use external PSKs that adhere to the following requirements:
+
+1. Each PSK MUST be at least 128-bits long unless the TLS handshake is being used with a separate key
```

Comment:
Should we also specify the input source? 128 bits chosen from a small space meets the criteria but doesn't help against exhaustive search.
```suggestion
1. Each PSK MUST be derived from at least 128 of entropy and MUST be at least 128-bits long unless the TLS handshake is being used with a separate key
```